Vizzuality/heco-invest
# feat: Finalisation of klab submit_contexts job

Final implementation of `Klab::SubmitContextsJob` including all tests.

## Testing instructions

Submit context to Klab of any project and verify that:
 - submits gets truly send to BC3 (no Faraday exceptions)
 - `Klab::PollImpactDemandsJob` jobs get enqueued with appropriate ticket numbers for all impact levels
 
You can also verify that job gets rerun in case BC3 is temporally down for whatever reason. Unfortunately this is a difficult to simulate at reality so I hope unit tests in this case are enough.

## Tracking

https://vizzuality.atlassian.net/browse/LET-1353
